src/online_managers.rs

```diff
@@ -262,15 +263,11 @@ impl<'a> Authenticator<'a> {
         }
     }
 
-    pub fn is_etebase_server(&self) -> Result<bool> {
+    pub fn validate_server(&self) -> Result<()> {
         let url = self.api_base.join("is_etebase/")?;
         let res = self.client.get(url.as_str())?;
-        if res.status() == 404 {
-            return Ok(false);
-        }
```

Comment:
What is the reason behind this? Why is a 404 different from any other error? If that's really an important distinction, the method should probably return a custom type that makes it more descriptive.

Reply:
Ah, that makes sense. I changed it so the functionality stays the same, but the method is still moved to a more obvious place and has a more descriptive name.